redash/query_runner/activedata.py

```diff
@@ -0,0 +1,180 @@
+import json
+import logging
+from urlparse import urlparse
+
+import requests
+# from mo_dots import wrap, listwrap
+# from mo_logs import Except
+# from mo_logs.exceptions import ERROR
+
+from redash.query_runner import TYPE_INTEGER, TYPE_STRING, TYPE_FLOAT, BaseSQLQueryRunner, register
+from redash.utils import JSONEncoder
+
+#originally from: https://github.com/klahnakoski/ActiveData-redash-query-runner/blob/c0e7286c09c6f1eb6746a6c7cca581bea79f4757/active_data.py
+
+logger = logging.getLogger(__name__)
+
+if not unicode:
+    unicode = str
```

Comment:
Looking at the built in query runners in redash and knowing that Python 3 doesn't have the `unicode` keyword, I'm assuming Python 2.x is the target.